unittests/SIL/SILBitfieldTest.cpp

```diff
@@ -0,0 +1,109 @@
+//===--- SILBitfieldTest.cpp ----------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+#include "gtest/gtest.h"
+
+
+namespace swift {
+
+class BasicBlockBitfield;
+
+// Fake SILFunction and SILBasicBlock.
+
+struct SILFunction {
```

Comment:
@eeckstein this is my favorite part of this commit! We need to do this more often!